### DISCUSSION TITLE
Enforcing Redirect for Trailing Slash Behavior in TanStack Start

### DISCUSSION THREAD

**Discussion by thefrana** [TanStack/router#2870] [Q&A]
Hi,

I came across the documentation [here](https://tanstack.com/router/latest/docs/framework/react/api/router/RouterOptionsType#trailingslash-property) regarding the trailingSlash property. I am using TanStack Start.

While it does handle URL construction, it seems that with the never option, accessing a URL like /some/path/ does not trigger a redirect to /some/path.

Is there a way to enforce such a redirect?

Thanks in advance!

**Comment by schiller-manuel** [comment]
can you please create a minimal example to showcase the issue?
